tests/testthat/test-abundance_normalization.R

```diff
@@ -0,0 +1,348 @@
+pxl_file <- system.file("extdata/five_cells",
+                        "five_cells.pxl",
+                        package = "pixelatorR")
+se <- ReadMPX_Seurat(pxl_file)
+
+test_that("NormalizeMethods work as expected", {
+  expect_no_error(
+    NormalizationMethod.dsb(LayerData(se, "counts"),
+                            isotype_controls = c("mIgG1", "mIgG2a", "mIgG2b")))
+  expect_no_error(
+    NormalizationMethod.clr(LayerData(se, "counts")))
+})
+
+test_that("NormalizeMPX works as expected", {
+
+  # Valid input
+
+  expect_no_error(
+    NormalizeMPX(LayerData(se, "counts")))
+  expect_no_error(
+    NormalizeMPX(se[["mpxCells"]]))
+  expect_no_error(
+    norm_data_clr <- NormalizeMPX(se, method = "clr"))
+
+  expect_equal(LayerData(norm_data_clr),
```

Comment:
Maybe it's enough to use a small slice of the data for this test so that it becomes more readable.

Reply:
Good point :)